linkerd/docs/namer.md

```diff
@@ -419,3 +419,52 @@ Key | Required | Description
 prefix | yes | Tells linkerd to resolve the request path using the marathon namer.
 zkPath | yes | The ZooKeeper path of a leader group. This path can be multiple path segments long. The namer resolves to the address stored in the data of the leader.
 
+<a name="rewritingNamers"></a>
+## Rewriting Namers
+
+In addition to service discovery namers, linkerd supplies a number of utility
+namers. These namers assist in path rewriting when the transformation is more
+complicated than just prefix substitution.
```

Comment:
Maybe note that these namers don't have to be included in the `namers:` list in your config file.  Since they are on the classpath and referenced with the `/$/` syntax, they will be automatically available.